document-generator-accounts/pom.xml

```diff
@@ -26,13 +26,15 @@
 
         <!--Dependencies-->
         <spring-boot-start-web.version>2.0.4.RELEASE</spring-boot-start-web.version>
-        <api-sdk-java.version>3.0.0-rc1</api-sdk-java.version>
+        <api-sdk-java.version>3.6.0-rc1</api-sdk-java.version>
```

Comment:
I think the latest skd is `<api-sdk-java.version>3.8.0-rc1</api-sdk-java.version>`

Reply:
The SDK updates so quickly. I'll need to test again on the update, which I will do.